.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,15 +10,21 @@ vulnerabilities.
 -->
 
 #### What it does
+
 <!-- Include relevant issues and describe how they are addressed. -->
 
 #### How to test
+
 <!-- Explain how a reviewer can reproduce a bug, test new functionality or verify performance improvements. -->
 
+#### Follow-ups
+
+<!-- Please list potential follow-up work, including know issues, possible future work, identified technical debt and potentially introduced technical debt. If the PR introduces technical debt, specify the reason, why this is acceptable. Please create tickets and link them here. Please use the label "technical debt" for new issues when it applies. -->
+
 #### Review checklist
 
-- [ ] As an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)
+-   [ ] As an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)
```

Comment:
minor: please revert.

Reply:
This is auto formatted this way when I save. I do not know where exactly this is configured, but I did not adapt this. So I guess the changes reflect the standard formatting?

Reply:
@JonasHelming I don't believe it is the standard formatting (confirmed locally on a fresh setup and with gitpod). In addition I believe it goes against most styleguides for markdown including rules from [markdownlint](https://github.com/DavidAnson/markdownlint/blob/v0.29.0/doc/md030.md) and the google style guide. It might be possible you have user settings that are incorrectly adding spacing after the list?

Reply:
Yes, you are probably right, I did not yet find the setting, but I have reverted the changes in this PR. Thank you!